src/app/sign-in.component.html

```diff
@@ -0,0 +1,10 @@
+<h2>Sign In</h2>
+Loaded: {{isLoaded}}
```

Comment:
I'm not sure what's up with that.

This modification to the original login component works (clicking the button changes the text):

```
diff --git a/src/app/login.component.html b/src/app/login.component.html
index 39a5efd..a60c0ab 100644
--- a/src/app/login.component.html
+++ b/src/app/login.component.html
@@ -1,4 +1,5 @@
-<h2>Log In</h2>
+<h2>Log In {{salutation}}</h2>
+<button (click)="change()">Change Salutation</button>
 <div *ngIf="user; else loggedout">
   <span>
     Logged in as {{user.name}} with permission level
diff --git a/src/app/login.component.ts b/src/app/login.component.ts
index 25a7131..ec4a947 100644
--- a/src/app/login.component.ts
+++ b/src/app/login.component.ts
@@ -15,6 +15,11 @@ export class LoginComponent implements OnInit {
 
   user: User;  // currently logged in, may be undefined
   login: User = new User();  // form data
+  salutation = "And Welcome";
+
+  change(): void {
+    this.salutation = "To AoU";
+  }
 
   ngOnInit(): void {
     this.userService.getLoggedInUser()
```